mynotes/Dockerfile

```
@@ -8,3 +8,5 @@ RUN npm install
 EXPOSE 3000
 
 CMD ["npm","start"]
+
+hello 
```

Hadolint (2.14.0)

[error] unexpected 'h'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)